daemon/nextbox_daemon/shield.py

```diff
@@ -15,7 +15,8 @@ def button_press_handler():
         return
     if shield.last_button_press + timedelta(seconds=2) > datetime.now():
         log.info("starting soft reset")
-        CommandRunner("nohup /usr/bin/nextbox-soft-reset.sh")
+        shield.set_led_state("soft-reset")
+        CommandRunner("nohup /usr/bin/nextbox-soft-reset.sh", block=True)
```

Comment:
do we really need `nohup` here, the nextbox-daemon itself should not be affected by the soft reset - given the issues we had with `nohup` I would suggest to avoid it if not strictly needed

Reply:
done